frontend/src/app/tasks/hooks/useTaskTypes.ts

```diff
@@ -0,0 +1,36 @@
+import { useQuery } from '@tanstack/react-query'
+import { useAuth } from 'react-oidc-context'
+
+type Task = {
+  slug: string
+  name: string
+  active: boolean
+}
+
+// Temporary disable so we don't need to change the fetch api for now.
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const fetchTasks = (token: string): Promise<Array<Task>> => {
```

Comment:
Same nit here: it might be clearer to name this `fetchTaskTypes` to avoid confusion between a task type and a task later on